frontend/.storybook/preview.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Theme } from "./../packages/core/src/AppProvider/themes";
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
```

Comment:
comment what the intent is here

Reply:
I'll actually remove this for now since we aren't really using it